### PR TITLE
Temperature log example

### PR DESCRIPTION
**Pull request by gabbapeople**
Update the case studies list with the microSD logging example.

### CODE REVIEW

**Comment by nkrkv** [comment]
Also, please, remove the “a” article from the article title. Titles use the [headlinese](https://en.wikipedia.org/wiki/Headlinese) style.

**Comment by gabbapeople** [comment]
Check it pls @nkrkv

**Comment by gabbapeople** [comment]
Check it pls @nkrkv

File: docs/guide/sd-log-example/README.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+You can move data values to the spreadsheet in different ways:
+
+- Select all the data from the `temp-log.txt` file by pressing CTRL+A in your
+  text editor. Copy what you’ve selected by pressing CTRL+C. Create a new Google
+  Spreadsheet document, select the first cell in the table and press CTRL+V.
+- Create a new Google Spreadsheet document. Hit `File -> Import.. -> Upload`.
+  Select the `temp-log.txt` file from the microSD card or drag-and-drop it into
+  the field. Select the `Tab` separator type and import the data.
+
+![Spreadsheet](./analyze-log-2.jpg)
```
Comment (nkrkv) [inline docs/guide/sd-log-example/README.md]: Not a photo, prefer PNG

File: docs/guide/sd-log-example/README.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+- Hit `CUSTOMIZE -> Chart & axis title`.
+- Input the titles for the vertical and horizontal axes. We named them
+  `Temperature, C` and `Time`.
+- Set the minimum and maximum bounds for the vertical and horizontal axes. We
+  set the `(10,90)` bound for the `Temperature, C` axis and `(0,3000)` for the
+  `Time` axis.
+- You can change the color of the line by hitting `Series` and choosing the
+  color.
+
+![Graph](./analyze-log-3.jpg)
```
Comment (nkrkv) [inline docs/guide/sd-log-example/README.md]: Not a photo, prefer PNG

File: docs/guide/sd-log-example/README.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+the IDE.
+
+## Conclusion
+
+With the microSD card, you can store a significant amount of data that can not
+be stored in the memory of the controller. You can use any other sensor or even
+several different sensors to log and observe a physical process of your choice.
+
+To improve a logger, use some [`led`](/libs/xod/common-hardware/led/) and
+[`delay`](/libs/xod/core/delay) nodes linked to `ERR` and `DONE` outputs so that
```
Comment (nkrkv) [inline docs/guide/sd-log-example/README.md]: Absolutize node URLs

**Comment by gabbapeople** [comment]
@nkrkv Check it pls

**Pull request merged**